### PR TITLE
Mission critical phase1

### PR DESCRIPTION
**Pull request by Mramcha**
Just added all README.mds with some links.

### CODE REVIEW

**Comment by Mramcha** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Mramcha** [comment]
recheck

File: data-platform/mission-critical-database/Disaster Recovery/README.md
Comment (AlexanderHodicke) [inline data-platform/mission-critical-database/Disaster Recovery/README.md:1]: disaster-recovery folder name

File: data-platform/mission-critical-database/High Availability/README.md
Comment (AlexanderHodicke) [inline data-platform/mission-critical-database/High Availability/README.md:1]: high-availability

**Review by AlexanderHodicke — CHANGES_REQUESTED** [review]

**Comment by Mramcha** [comment]
recheck

File: data-platform/mission-critical-database/High Availability/README.md
Comment (Mramcha) [reply data-platform/mission-critical-database/High Availability/README.md:1]: done

File: data-platform/mission-critical-database/Disaster Recovery/README.md
Comment (Mramcha) [reply data-platform/mission-critical-database/Disaster Recovery/README.md:1]: done

File: data-platform/mission-critical-database/Data Protection/README.md
[the remark this answers is not in the document]
Comment (Mramcha) [reply data-platform/mission-critical-database/Data Protection/README.md:1]: done.

**Review by AlexanderHodicke — APPROVED** [review]

**Pull request merged**